src/modules/login_user/app/login_user_presenter.ts

```diff
@@ -0,0 +1,22 @@
+import { Environments } from '../../../shared/environments'
+import { LambdaHttpRequest, LambdaHttpResponse } from '../../../shared/helpers/external_interfaces/http_lambda_requests'
+import { LoginUserController } from './login_user_controller'
+import { LoginUserUsecase } from './login_user_usecase'
+
+const repo = Environments.getUserRepo()
+const usecase = new LoginUserUsecase(repo)
+const controller = new LoginUserController(usecase)
+
+export async function loginUserPresenter(event: Record<string, any>) {
```

Comment:
we're not using lambda for now, the presenter will receive a HttpRequest(IRequest) and return just a HttpResponse, no lambda